### PR TITLE
libvirt-python 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by vicamo**
Signed-off-by: You-Sheng Yang (楊有勝) <vicamo@gmail.com>

### CODE REVIEW

File: libvirt-python.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  homepage "http://www.libvirt.org"
+
+  stable do
+    url "http://libvirt.org/sources/python/libvirt-python-1.3.1.tar.gz"
+    sha256 "7143b922a9bd66a5e199b13316fa4266cbffc23db01c99bce37216a1eb9118f7"
+  end
+
+  head do
+    url "git://libvirt.org/libvirt-python.git"
+  end
```
Comment (bfontaine) [inline libvirt-python.rb]: You can use `head "git://libvirt.org/libvirt-python.git"`.

File: libvirt-python.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class LibvirtPython < Formula
+  desc "Python bindings for libvirt"
+  homepage "http://www.libvirt.org"
+
+  stable do
+    url "http://libvirt.org/sources/python/libvirt-python-1.3.1.tar.gz"
+    sha256 "7143b922a9bd66a5e199b13316fa4266cbffc23db01c99bce37216a1eb9118f7"
+  end
```
Comment (bfontaine) [inline libvirt-python.rb]: You can put these at top-level, i.e. without a `stable do` block.

**Comment by vicamo** [comment]
pushed new commit https://github.com/vicamo/homebrew/commit/4dc43beeb24003c497cc271dd8dc35d7b473511e to address review comments.

File: libvirt-python.rb (hunk 1 above)
Comment (vicamo) [inline libvirt-python.rb]: addressed in https://github.com/vicamo/homebrew/commit/4dc43beeb24003c497cc271dd8dc35d7b473511e

File: libvirt-python.rb (hunk 2 above)
Comment (vicamo) [inline libvirt-python.rb]: addressed in https://github.com/vicamo/homebrew/commit/4dc43beeb24003c497cc271dd8dc35d7b473511e

**Comment by mikemcquaid** [comment]
Can this be installed with `pip`?

**Comment by DomT4** [comment]
Looks like it: https://pypi.python.org/pypi/libvirt-python/1.3.1

File: libvirt-python.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  desc "Python bindings for libvirt"
+  homepage "http://www.libvirt.org"
+  url "http://libvirt.org/sources/python/libvirt-python-1.3.1.tar.gz"
+  sha256 "7143b922a9bd66a5e199b13316fa4266cbffc23db01c99bce37216a1eb9118f7"
+  head "git://libvirt.org/libvirt-python.git"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "libvirt"
+  depends_on "python"
```
Comment (DomT4) [inline libvirt-python.rb]: Python dependencies should almost never be hard. Please see the Python documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md#python-declarations

**Comment by vicamo** [comment]
Close to use libvirt-python from pip in virt-manager.

**Pull request closed without merging**